support-frontend/.flowconfig

```diff
@@ -5,6 +5,11 @@
 .*target/.*
 .*/__tests__/.*
 
+[untyped]
```

Comment:
What is different about this compared to the ignore list above?

Reply:
I believe if it's ignored you can't import it, whereas if it's untyped it's given the type 'any'